projects/vdk-plugins/vdk-kerberos-auth/tests/jobs/test-job/20_async_test.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import asyncio
+import logging
+
+from vdk.api.job_input import IJobInput
+
```

Comment:
Did you run the modified unit tests with the source code version before the fix ? 

I very much recommend when writing a test that verifies a fix  to make sure you run it with the previous version and make certain the test *fails* and after the fix is applied the test passes. This gives you very high certainty that the root cause, the problem and the fix have been correctly identified. 

I suggest you revert your change and try to run the test. It should fail if the root cause was correct.

Reply:
Very good catch. Changed the test, and verified it breaks without the changes.
Thank you